### ISSUE TITLE
Broken Chart/Song reloading

### ISSUE THREAD

**Issue by WillyTheModder** [ShadowMario/FNF-PsychEngine#2862]
### Describe your bug here.

Hi,

I was trying to chart stuff in Psych Engine, so I did what I needed to do when making a custom song, I went to Roses (which I choose when making a new ,json for charting), renamed Roses to my song name, clicked reload song, and it would only show 0 / 0.

I restarted the game, nothing. And I tested it with a song I already charted, same problem, even wouldn't reload the chart that I did in fact save and did work earlier, before this.

I don't know what happened, but has to be fixed, somehow.

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

I didn't edit anything.

**Comment by ShadowMario** [comment]
Press enter to get in the song then go back to Chart Editor.
If it's still not loading the song then your song is corrupted or missing the Inst.ogg file inside assets/songs/your-song-name/

**Comment by WillyTheModder** [comment]
1. I did that just now, and it is still not working.
2. It is supposed to only be an Inst.ogg file for me, cause that's how I am charting this song.

One thing to note, it automatically resets the stage from the school in Week 6 to the stage after I press 7 on Roses. The file can't be corrupted, since I tested this with another chart I made that I know would work earlier, and the same issue happened. It even wouldn't reload the chart which was in fact already there and completed.
